htdocs/comm/propal/card.php

```diff
@@ -642,7 +642,7 @@
 			if ($object->statut == $object::STATUS_VALIDATED) {
 				$db->begin();
 
-				$result = $object->cloture($user, GETPOST('statut', 'int'), GETPOST('note_private', 'restricthtml'));
+				$result = $object->signature($user, GETPOST('statut', 'int'), GETPOST('note_private', 'restricthtml'));
```

Comment:
Currently the user that set the status of proposal to "signed" by customer is saved into field   fk_cloture (for "closed as signed").
What will be the goal of having a second field ?
Is it to track a status signed by customer or signed by the company ?
Is it to add a second level of validation of proposal by a second user different than the one who validate the proposal  before sending it to the customer ?

Reply:
@eldy
Having a second field is needed to differenciate user who sign and user who close the commercial proposal (as we defined in issue 16062).

Reply:
I see. Thanks.